tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -27,7 +27,7 @@
                     <converters:BoolToVisibilityConverter x:Key="NegatedBoolToVisibilityConverter"  TrueValue="Collapsed" FalseValue="Visible" />
                     <converters:BoolToObjectConverter x:Key="BoolToGlyphConverter" TrueValue="&#xF0BD;" FalseValue="&#xF03F;"/>
                     <ControlTemplate x:Key="LinksTemplate">
-                        <StackPanel Orientation="Vertical" BorderBrush="{ThemeResource DividerStrokeColorDefaultBrush}" BorderThickness="{ThemeResource TopNavigationViewContentGridBorderThickness}">
+                        <StackPanel Orientation="Vertical">

```

Comment:
Nit: vertical is default so you could remove this if you want.

Reply:
Done.